packages/admin/blocks-admin/src/blocks/YouTubeVideoBlock.tsx

```diff
@@ -26,6 +42,8 @@ export const YouTubeVideoBlock: BlockInterface<YouTubeVideoBlockData, State, You
         return { ...state, autoplay: false, adminMeta: { route: previewCtx.parentUrl } };
     },
 
+    isValid: (state) => isValidYouTubeIdentifier(state.youtubeIdentifier),
```

Comment:
Can't the identifier be `optional` here?

Reply:
No cause then the block would not make any sense I guess. Also there is a default value.

Reply:
When you add an identifier and then clear the input field, the state of the identifier will be `undefined`. Furthermore, new blocks should be savable by default (see [docs](https://docs.comet-dxp.com/docs/blocks/block-best-practices#allow-saving-a-block-with-its-default-values)), and I don't think this works.

Reply:
You're right. I fixed it.